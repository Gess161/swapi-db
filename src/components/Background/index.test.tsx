import renderer from 'react-test-renderer';
import StarrySky from '.';

it('renders stars background with twinkling animation"', () => {
    const tree = renderer
        .create(<StarrySky />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});