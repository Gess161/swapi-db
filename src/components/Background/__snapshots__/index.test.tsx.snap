// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders stars background with twinkling animation" 1`] = `
<div
  className="sc-dkPtRN jEdA-dV"
>
  <div
    className="sc-bdvvtL gwERyl"
  />
  <div
    className="sc-gsDKAQ exBuuV"
  />
</div>
`;
